serving/src/test/java/ai/djl/serving/ModelServerTest.java

```diff
@@ -596,6 +600,16 @@ private void testListModels(Channel channel) throws InterruptedException {
                             .anyMatch(w -> expectedModel.equals(w.getModelName())));
         }
 
+        // Test pure JSON works
+        Map<String, Object> rawResult = JsonUtils.GSON.fromJson(result, Map.class);
```

Comment:
the result has been deserialized into `resp` object, can we just assert each fields are not null?

Reply:
I'm not sure that helps. The issue was that I changed the property name of the list from `models` to `wpcs`. This changes both the serialization in the code and deserialization in the test, so no explicit changes were necessary in the test. I wanted to use something more raw so that the names of the fields explicitly show up in our testing